extension/src/background_scripts/background.js

```diff
@@ -0,0 +1,11 @@
+// eslint-disable-next-line no-undefined
+chrome.runtime.onInstalled.addListener(function() {
```

Comment:
this should no longer report error on next push

Reply:
resolved